README.md

```diff
@@ -1,2 +1,33 @@
 # dataset-utils
 A collection of JavaScript functions to read and parse OpenActive Data Catalogs and Dataset Sites
+
+## Install
+
+Install using NPM:
+```
+$ npm install @openactive/dataset-utils --save
+```
+
+## Dependencies
+- [Axios](https://github.com/axios/axios)
+
+## Usage
+
+```js
+const getAllDatasetSiteUrls = require('@openactive/dataset-utils');
+const EXAMPLE_DATA_CATALOG_COLLECTION = 'https://openactive.io/data-catalogs/data-catalog-collection.jsonld';
+
+var datasetUrls = getAllDatasetSiteUrls(EXAMPLE_DATA_CATALOG_COLLECTION);
```

Comment:
Suggest the usage example shows the most comprehensive example.

Also this is an async method?

```suggestion
var datasetUrls = await getAllDatasets(EXAMPLE_DATA_CATALOG_COLLECTION);
```